modules/plugin/chowdsp_plugin_utils/SharedUtils/chowdsp_GlobalPluginSettings.cpp

```diff
@@ -13,6 +13,7 @@ GlobalPluginSettings::SettingsFileListener::SettingsFileListener (const juce::Fi
 
 void GlobalPluginSettings::SettingsFileListener::listenerFileChanged()
 {
+    const juce::ScopedLock sl (globalSettings.lock);
```

Comment:
I think we can get by without this one, since the lock happens first thing in `loadSettingsFromFile()`?

Reply:
Good call